UPGRADING.md

```diff
@@ -1,316 +1,277 @@
+| `BugsnagBreadcrumb` constructors                                   | Deprecated - no longer public API                                 |
+| `BugsnagBreadcrumbs`                                               | Deprecated - no longer public API                                 |
 
-### `BugsnagMetadata` class
 
-#### Renames
+### Events (`BugsnagCrashReport`)
 
-```diff
-ObjC: 
+`BugsnagCrashReport` is now called `BugsnagEvent` and contains all the data representing the error that's been captured for access and/or mutation in callbacks.
 
-- [BugsnagMetadata clearTabWithName:]
-+ [BugsnagMetadata clearMetadataInSection:]
+#### Additions
 
-- [BugsnagMetadata getTab:]
-+ [BugsnagMetadata getMetadata:]
+The following options have been added to the `BugsnagEvent` class: 
 
-+ [BugsnagMetadata addMetadataToSection:values:]
+| Property/Method                                                    | Usage                                                             |
+| ------------------------------------------------------------------ | ----------------------------------------------------------------- |
+| `unhandled`                                                        | Whether the error was detected automatically by Bugsnag or reported manually via notify/notifyError. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#unhandled) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#unhandled) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#unhandled)).
+| `originalError`                                                    | The original object that caused the error in your application. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#originalerror) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#originalerror) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#originalerror)).
+| `user` / `setUser:withEmail:andName` / `setUser(_:email:name)`     | The user of the app when the event occurred. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#setuser) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#setuser) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#setuser)).
 
-Swift:
+#### Changes
 
-- BugsnagMetadata.clearTab(name:)
-+ BugsnagMetadata.clearMetadata(section:)
+The `device`, `app`, `errors` and `threads` fields on the `BugsnagEvent` class are now structured classes, rather than `NSDictionary` types. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#the-bugsnagevent-class) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#the-bugsnagevent-class) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#the-bugsnagevent-class)).
```

Comment:
I'm not 100% sure these were available in `v5`

Reply:
Reworded so this is no longer implied.